rsmtool/reader.py

```diff
@@ -22,6 +22,47 @@
 from rsmtool.container import DataContainer
 
 
+
+def read_jsonlines(filename, converters=None):
+    """ Read jsonlines from a file. 
+    Normalize nested jsons with up to one level of nesting
+
+    Parameters
+    ----------
+    filename: str
+        Name of file to read
+    converters : dict or None, optional
+        A dictionary specifying how the types of the columns
+        in the file should be converted. Specified in the same
+        format as for `pd.read_csv()`.
+
+    Returns
+    -------
+    df : pandas DataFrame
+         Data frame containing the data in the given file.
+    """
+
+    df = pd.read_json(filename,
+                      orient='records',
+                      lines=True,
+                      dtype=converters)
+        
+    # let's check if we have nested columns
+    nested_cols = [c for c in df.columns if isinstance(df[c][0],dict)]
```

Comment:
What if the first row in the column is null?

Reply:
Can you give an example?